### PR TITLE
Feat/permit actions

### PR DESCRIPTION
**Pull request by majkic99**

### CODE REVIEW

File: test/utils-actions/utils-actions-tests.js [hunk 1]
```diff
@@ -348,6 +349,93 @@ const approveTokenTest = async () => {
...
+            proxy = await getProxy(senderAcc.address);
+        });
+        it('... should permit tokens for DSProxy to pull from eoa', async () => {
+            const snapshot = await takeSnapshot();
+            const network = hre.network.config.name;
+            const chainId = chainIds[network];
+            const wsteth = getAssetInfo('wsteth', chainId);
+            const wstethAddress = wsteth.address;
+
+            const daiPermitContract = await hre.ethers.getContractAt('IERC20Permit', wstethAddress);
```
Comment (yippee-ki-yay) [inline test/utils-actions/utils-actions-tests.js:370]: dai -> wsteth

File: test/morpho/aaveV3/morpho-aaveV3-setManagerBySig.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const { redeploy } = require('../../utils');
+const { morphoAaveV3SetManagerBySigTest } = require('./morpho-aaveV3-tests');
+
+describe('Morpho-Aave-V3-SetManagerBySig', function () {
+    this.timeout(80000);
+    before(async () => {
+        await redeploy('MorphoAaveV3SetManagerBySig');
+    });
+
+    it('... should test Morpho AaveV3 supply', async () => {
```
Comment (yippee-ki-yay) [inline test/morpho/aaveV3/morpho-aaveV3-setManagerBySig.js:10]: wrong test name

**Review by yippee-ki-yay — APPROVED** [review]

File: contracts/actions/compoundV3/CompV3AllowBySig.sol [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        return uint8(ActionType.STANDARD_ACTION);
+    }
+
+    //////////////////////////// ACTION LOGIC ////////////////////////////
+
+    function _allow(Params memory _params) internal returns (bytes memory) {
+        IComet(_params.market).allowBySig(_params.owner,_params.manager, _params.isAllowed, _params.nonce, _params.expiry, _params.v, _params.r, _params.s);
+        
+        ///@dev Every successful call to permit increases owners nonce by one. 
+        require (IComet(_params.market).userNonce(_params.owner) == _params.nonce + 1);
```
Comment (kjaredic) [inline contracts/actions/compoundV3/CompV3AllowBySig.sol:54]: is this check warranted?

**Review by kjaredic — APPROVED** [review]
lgtm

**Pull request closed without merging**